src/Performance/Minibench.purs

```diff
@@ -12,14 +12,14 @@ module Performance.Minibench
   , withUnits
   ) where
 
-import Prelude hiding (min,max)
+import Prelude hiding (min, max)
```

Comment:
Can you revert all the changes to this file just so that we minimize any conflicts with the upstream version in the future.